jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
 
+let orValue a x =
+  match x with
+  | None -> a
+  | Some x -> x
 
+let filter f x =
+  match x with
+  | None -> None
```

Comment:
Eh, not really 'common', but it is something I've used before myself, would be nice to include it even if it is not really 'standard'.